gateway-protocol/src/main/proto/gateway.proto

```diff
@@ -84,69 +84,69 @@ message CompleteJobRequest {
 message CompleteJobResponse {
 }
 
-message CreateWorkflowInstanceRequest {
-  // the unique key identifying the workflow definition (e.g. returned from a workflow
-  // in the DeployWorkflowResponse message)
-  int64 workflowKey = 1;
-  // the BPMN process ID of the workflow definition
+message CreateProcessInstanceRequest {
+  // the unique key identifying the process definition (e.g. returned from a process
+  // in the DeployProcessResponse message)
+  int64 processKey = 1;
+  // the BPMN process ID of the process definition
   string bpmnProcessId = 2;
   // the version of the process; set to -1 to use the latest version
   int32 version = 3;
   // JSON document that will instantiate the variables for the root variable scope of the
-  // workflow instance; it must be a JSON object, as variables will be mapped in a
+  // process instance; it must be a JSON object, as variables will be mapped in a
   // key-value fashion. e.g. { "a": 1, "b": 2 } will create two variables, named "a" and
   // "b" respectively, with their associated values. [{ "a": 1, "b": 2 }] would not be a
   // valid argument, as the root of the JSON document is an array and not an object.
   string variables = 4;
 }
 
-message CreateWorkflowInstanceResponse {
-  // the key of the workflow definition which was used to create the workflow instance
-  int64 workflowKey = 1;
-  // the BPMN process ID of the workflow definition which was used to create the workflow
+message CreateProcessInstanceResponse {
+  // the key of the process definition which was used to create the process instance
+  int64 processKey = 1;
```

Comment:
The name `processKey` could be confusing in combination with `bpmnProcessId`. Maybe, we could use `processDefinitionKey` instead.

Reply:
Agree on that. This confused my always in camunda 7 processId vs processKey

Reply:
I renamed it to `processDefinitionKey`